tests/testthat/helper-indexing.R

```diff
@@ -0,0 +1,24 @@
+mm <- function(...) {
```

Comment:
this is mm(). It takes a vector and makes into a matrix. vector_to_matrix?

Reply:
How about `vector_to_square_matrix`?

Reply:
Thanks!